src/ResourceManager/Automation/Commands.Automation/packages.config

```diff
@@ -4,8 +4,8 @@
   <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Common.Authentication" version="1.5.2-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Automation" version="0.50.2-prerelease" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Resources" version="2.18.11-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Automation" version="2.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Resources" version="2.18.7-preview" targetFramework="net45" />
```

Comment:
looks like incorrectly updated during merge within team, Fixed to be back to 2.18.11. 
Sure 2.18.13 should be fine for us